backend/azure-functions/src/IsSupervisor/index.ts

```diff
@@ -0,0 +1,27 @@
+import { PassedAuthenticationResult, authenticate } from '../Auth/facebook-auth';
+import { Context, HttpRequest } from 'azure-functions-ts-essentials';
+
+const allowedUserIds: string[] = [
+  // TODO: add users
+];
+
+const IsSupervisor = async function (context: Context, req: HttpRequest): Promise<void> {
+  const authenticatedUser: PassedAuthenticationResult = await authenticate(req, context, true) as PassedAuthenticationResult;
+  if (!(authenticatedUser instanceof PassedAuthenticationResult)) return;
+
+  if (allowedUserIds.indexOf(authenticatedUser.userId) > -1) {
```

Comment:
I guess that this logic (allowedUserIds array + call to auth + check if user in list) should be extracted so can be reused by NewReport.